engine/graphics/src/dmsdk/graphics/graphics.h

```diff
@@ -87,6 +87,13 @@ namespace dmGraphics
      */
     typedef struct VertexDeclaration* HVertexDeclaration;
 
+    /*#
+     * Vertex stream declaration handle
+     * @typedef
+     * @name HVertexStreamDeclaration
+     */
+    typedef uintptr_t HVertexStreamDeclaration;
```

Comment:
The reason the HVertexDeclaration uses a struct name, is so that the debugger will automatically convert it to that type, and you'll see the values directly.

If using a void*, you'll have to cast it yourself.
I see no downside to using the "struct VertexStreamDeclaration" here?